serverless/interacting-serverless-apps.adoc

```diff
@@ -6,7 +6,8 @@ include::modules/serverless-document-attributes.adoc[]
 
 toc::[]
 
-This section provides information about various tasks that can be performed once you have successfully created a serverless application.
+This section provides information about various tasks that can be performed once you have successfully created a serverless application using Knative Services.
 
 include::modules/verifying-serverless-app-deployment.adoc[leveloffset=+1]
 include::modules/interacting-serverless-apps-http2-gRPC.adoc[leveloffset=+1]
```

Comment:
@markusthoemmes do we actually want to include this?
In the [1.2.0 RN](https://docs.openshift.com/container-platform/4.2/serverless/serverless-release-notes.html#known-issues-1-2-0_serverless-release-notes) we say:
"gRPC and HTTP2 do not work against routes. This is a known limitation of OpenShift routes."

Reply:
Yes, IIRC this contains the workaround to still make it work. It might be slightly out of date though as `istio-system` references need to reference `knative-serving-ingress` now.

Reply:
Does the `istio-system` reference in https://docs.openshift.com/container-platform/4.2/serverless/serverless-release-notes.html#known-issues-1-2-0_serverless-release-notes need to be updated to `knative-serving-ingress` too? That's what my Jira issue that I couldn't delete earlier was about :slightly_smiling_face:

Reply:
Nope, that one is fine as it points at the state of version 1.1 and that reference was correct back then. We're good I think.